### DISCUSSION TITLE
Can Mathjs be used for integrals or definite integrals

### DISCUSSION THREAD

**Discussion by wanglifeng95** [josdejong/mathjs#3210] [Q&A]
Mathjs is  very useful, but how should I use it to calculate integrals or definite integrals?

**Comment by josdejong** [comment]
Mathjs has no built in support for integrals. It would be great to have that though :)

There was an initiative here, an extension, that can be interesting: https://github.com/josdejong/mathjs/discussions/2748

**Comment by wanglifeng95** [comment]
您好，我已收到来信，会尽快给予你回复。

**Reply by josdejong** [reply]
Can you please edit your message and translate it to English?

**Reply by wanglifeng95** [reply]
Sorry, there was an automatic reply from above.
Sympy is a computer algebra system written in pure Python. I like mathjs very much. I really look forward to  Mathjs integrating integral functions like Sympy does. 
[mathjs-simple-integral] appeared in 2018, and 6 years have passed. It's a pity that it just gave up like this

**Reply by josdejong** [reply]
It's all open source, anyone can pick it up again, including you 😄 .

**Reply by wanglifeng95** [reply]
If I had that ability, I would really be willing to help it make up for the last important puzzle piece.
